apps/wallet/src/ui/app/experimentation/experiments.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+export enum EXPERIMENTS {
```

Comment:
It's unclear what this is here for

Reply:
It was intended to use in the next PR, I just removed it from this PR.